jobs/pulse3d/pulse3d-worker/src/main.py

```diff
@@ -133,25 +139,27 @@ async def process(con, item):
                     for arg_name in ("stiffness_factor", "inverted_post_magnet_wells", "well_groups")
                 }
 
+                # well groups should always be added regardless of reanalysis
+                well_groups = plate_recording_args.get("well_groups")
                 use_existing_time_v_force = re_analysis and not any(plate_recording_args.values())
 
                 logger.info("Starting pulse3d analysis")
                 if use_existing_time_v_force:
                     # if any plate recording args are provided, can't load from data frame since a re-analysis is required to recalculate the waveforms
                     logger.info(f"Loading previous time force data from {parquet_filename}")
                     recording_df = pd.read_parquet(parquet_path)
-                    # well groups should always be added regardless of reanalysis
-                    well_groups = plate_recording_args.get("well_groups", None)
 
                     try:
                         recording = PlateRecording.from_dataframe(
-                            os.path.join(tmpdir, filename), df=recording_df, well_groups=well_groups
+                            os.path.join(tmpdir, analysis_filename),
+                            recording_df=recording_df,
+                            well_groups=well_groups,
                         )
                         recordings = list(recording)
                     except:
                         # If a user attempts to perform re-analysis on an analysis from < 0.25.2, it will fail
                         # because the parquet file won't have the raw data columns, so need to re-analyze
-                        logger.info(
+                        logger.exception(
```

Comment:
Can we also log the exception traceback here?

Reply:
Or at least just the error itself

Reply:
logger.exception auto includes the traceback

Reply:
Wow I did not realize that